files/icons/material-nf.toml

```diff
@@ -131,3 +131,4 @@ weather_thunder = "\ue31d" # nf-weather-thunderstorm
 weather_thunder_night = "\ue32a" # nf-weather-night_alt_thunderstorm
 webcam = "\U000f0567" # nf-md-video
 xrandr = "\U000f037a" # nf-md-monitor_multiple
+hueshift = "\uf400" # nf-oct-light_bulb
```

Comment:
I'd recommend `f0336` `nf-md-lightbulb_outline`. We prefer the md (material design) fonts in the material nerd fonts file